metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1687,33 +1687,35 @@ class MetalsLanguageServer(
 
     try {
       val sourceToIndex0 = sourceToIndex(source, targetOpt)
-      val reluri = source.toIdeallyRelativeURI(sourceItem)
-      val input = sourceToIndex0.toInput
-      val symbols = ArrayBuffer.empty[WorkspaceSymbolInformation]
-      SemanticdbDefinition.foreach(input) {
-        case SemanticdbDefinition(info, occ, owner) =>
-          if (WorkspaceSymbolProvider.isRelevantKind(info.kind)) {
-            occ.range.foreach { range =>
-              symbols += WorkspaceSymbolInformation(
-                info.symbol,
-                info.kind,
-                range.toLSP
-              )
+      if (sourceToIndex0.exists) {
```

Comment:
This is actually only an issue if ammonite failed to generate the scala files, which also means we should watch the `.ammonite` directory, since they can be created after indexing :thinking:

Reply:
Actually, it didn't fail to generate, ammonite just differently generates path if `^` is used, which should not normally be an issue, but we should leave the guard just in case: I will change the tests a bit so that it's more natural path and not `^.backToTheSameDirectory`